benchmark/fs/bench-stream-via-pipeline.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+const common = require('../common.js');
+const tmpdir = require('../../test/common/tmpdir');
+const path = require('path');
+const fsp = require('fs/promises');
+const fs = require('fs');
+const stream = require('stream');
+
+const KB = 2 ** 10;
+const MB = 2 ** 20;
+
+const configs = {
+  n: [100],
+  fileSize: [
+    1 * KB,
+    1 * MB,
+    10 * MB,
+  ],
+};
+
+tmpdir.refresh();
+const sourceRoot = path.resolve(tmpdir.path, 'source');
+const destRoot = path.resolve(tmpdir.path, 'dest');
+
+const bench = common.createBenchmark(main, configs);
+
+async function main(conf) {
+  const { fileSize } = conf;
+  // prepare temp files
+  const content = 'a'.repeat(fileSize);
+
+  const sourceName = `${sourceRoot}-${fileSize}`;
+  const destName = `${destRoot}-${fileSize}`;
+
+  const testFixtures = await Promise.all(
+    new Array(conf.n).fill(null).map((_, i) => prepareFixture(i))
+  );
+
+  bench.start();
+
+  await Promise.all(
+    testFixtures.map(
+      (testFixture) => {
+        return new Promise((resolve) => {
+          stream.pipeline(testFixture[0], testFixture[1], () => {
+            resolve();
+          });
+        });
```

Comment:
Shouldn't we use 
```js
const { pipeline } = require('stream/promises');
```
and
```suggestion
        return pipeline(testFixture[0], testFixture[1]);
```